integration/configs/browserCompatibilityTest/video_test.config.json

```diff
@@ -0,0 +1,56 @@
+{
+  "`": 1,
+  "name": "video_test.config.json %ts",
+  "grids": [],
+  "tests": [
+    {
+      "name": "Video Test %ts",
+      "description": "Video Test",
+      "tupleSize": 2,
+      "noOfThreads": 1,
+      "testImpl": "VideoTest.js",
+      "payload": {
+        "url": "http://localhost:8080/?earlyConnect=1",
+        "port": 30002,
+        "retry": 2
+      }
+    }
+  ],
+  "clients": [
```

Comment:
Question: Any reason why we do not run against Firefox here?
Saucelabs flags issue may be 😅

Reply:
Video tests will not run against Firefox (FF) because there is an issue with H.264 encoding on Windows. Also, the FF tests will not run MacOs due to some stability issues so need to investigate that more and need to include the tests once resolved.